packages/drivers/odsp-driver/src/odspFluidFileLink.ts

```diff
@@ -61,11 +61,12 @@ export function encodeOdspFluidDataStoreLocator(locator: OdspFluidDataStoreLocat
 function decodeOdspFluidDataStoreLocator(
 	encodedLocatorValue: string,
 	siteOriginUrl: string,
+	requireFluidSignature: boolean = true,
 ): OdspFluidDataStoreLocator | undefined {
 	const locatorInfo = new URLSearchParams(fromBase64ToUtf8(encodedLocatorValue));
 
 	const signatureValue = locatorInfo.get(fluidSignatureParamName);
-	if (signatureValue !== "1") {
+	if (requireFluidSignature && signatureValue !== "1") {
```

Comment:
Please make sure backwards compatibility works as expected. There are 2 cases:
1. Existing call sites can still call this new API without `requireFluidSignature`. The behavior would now be different - It won't go into this if condition anymore. Is that okay?
2. New call sites will call into the old API without `requireFludSignature`. The call site should account for the old behavior.

Reply:
requireFluidSignature is true by default so in fact backward compatibility is retained.

Reply:
You are right. I somehow thought it was false :-).